packages/home_widget/example/android/app/src/main/AndroidManifest.xml

```diff
@@ -61,8 +61,6 @@
                 <action android:name="es.antonborri.home_widget.action.BACKGROUND" />
             </intent-filter>
         </receiver>
-        <service android:name="es.antonborri.home_widget.HomeWidgetBackgroundService"
```

Comment:
Given that people should/need to remove this I would propose to:

- Keep `HomeWidgetBackgroundService`
- Mark it as Deprecated. Hinting that adding it to the Manifest could be omitted

That way this update could be a non breaking one and the full removal could be done with the next update that would also include a breaking change on iOS migrating to Swift Package Manager